network/domains-and-dns/how-to/add-external-domain.mdx

```diff
@@ -0,0 +1,56 @@
+---
+meta:
+  title: How to add an external domain to DNS
+  description: This page explains how add an external domain to DNS
+content:
+  h1: How to add an external domain to DNS
+  paragraph: This page explains how add an external domain to DNS
+dates:
+  validation: 2022-10-31
+  posted: 2022-10-31
+---
+
+[Scaleway Domains and DNS](https://www.scaleway.com/en/dns/) is a managed DNS service that allows you to easily configure the DNS zones of your domains. It provides support for queries via both IPv4 and IPv6 and supports all common types of DNS records.
+
+Scaleway Domains and DNS is currently available to manage DNS zones for external domains (ones created at external registrars). 
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com) 
+</Message>
+
+1. Click **DNS** in the **Network** section of the [Scaleway Console](https://console.scaleway.com) side menu. The DNS zone dashboard displays. If you have not created a zone yet, the creation page displays:
+      <Lightbox src="scaleway-dns-interface.webp" alt="" />
+2. Enter the external domain name in the box and click the search icon to verify the domain exists. Once verified, the following message displays:
+      <Lightbox src="scaleway-dns-domain-validated.webp" alt="" />
+3. Click **Next** to validate your domain name. The validation instructions display. Add a `TXT`record to the root zone of the domain at your current registrar to validate the domain ownership.
+      <Message type="note">
+        `TXT` means text record and it is used to carry machine-readable data such as information for automated domain validation.
```

Comment:
Either:
`TXT` means text record and **is** used...
`TXT` means text record. **It is** used...